src/popup/pages/Staking/StakeNode.js

```diff
@@ -11,6 +11,8 @@ import Clock from "../../component/Clock";
 import "./index.scss";
 import loadingCommon from "../../../assets/images/loadingCommon.gif";
 import noHistory from "../../../assets/images/noHistory.png";
+import shuffle from 'lodash/shuffle';
```

Comment:
Shouldn't we use `lodash-es`?

Reply:
Has lodash not fixed itself yet?

Reply:
Building with `lodash/shuffle` is 10kb smaller than `lodash-es/shuffle` :shrug: